test/fixtures/test-runner/output/describe_it.js

```diff
@@ -290,12 +290,12 @@ describe('subtest sync throw fails', () => {
 });
 
 describe('describe sync throw fails', () => {
-  it('should not run', () => {});
+  it('should run', () => {});
```

Comment:
isn't it a change in behavior?

Reply:
yes, it is. it makes sense though, and I think it can be considered as a fix, so not really a semver-major